packages/docs/actions/ResetValidation.yaml

```diff
@@ -0,0 +1,102 @@
+# Copyright 2020-2021 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/actions.yaml.njk
+  vars:
+    pageId: ResetValidation
+    pageTitle: ResetValidation
+    filePath: actions/ResetValidation.yaml
+    types: |
+      ```
+      (void): void
+      (blockId: string): void
+      (blockIds: string[]): void
+      (params: {
+        blockId?: string|string[],
+        regex?: string|string[],
+      }): void
+      ```
+    description: |
+      The `ResetValidation` action is used to reset validation flags on input fields, usually after a `Validation` flag was called to remove input error and warning indications from fields.
```

Comment:
Remove: ", usually after a `Validation` flag was called to remove input error and warning indications from fields.", I think the next sentence explains things better